fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -123,13 +202,42 @@ Config parseConfiguration(final String[] args) {
             config.setRdfExtension(cmd.getOptionValue('x', DEFAULT_RDF_EXT));
             config.setRdfLanguage(cmd.getOptionValue('l', DEFAULT_RDF_LANG));
 
-        } catch (ParseException e) {
+        } catch (final ParseException e) {
             printHelp("Error parsing args: " + e.getMessage());
         }
         return config;
     }
 
     /**
+     * This method writes the configuration file to disk
+     *
+     * @param args to be persisted
+     */
+    private void saveConfig(final String[] args) {
+        final File configFile = new File(System.getProperty("java.io.tmpdir"), CONFIG_FILE_NAME);
```

Comment:
Is `java.io.tmpdir` a good place to store the config file?  Won't that be removed when the computer is rebooted?  Maybe it's a good fallback location for the config file (since we can be pretty sure it will exist and we can write a file there), but maybe some other user-configurable location would be better?